assets/javascripts/discourse/initializers/discourse-calendar.js

```diff
@@ -329,15 +351,15 @@ function initializeDiscourseCalendar(api) {
     };
 
     let event = {
-      start: from.dateTime.toDate(),
+      start: from.dateTime.format("YYYY-MM-DD"),
```

Comment:
Is this going to drop the time of the event on the list view? I.e. this: 

<img width="762" alt="image" src="https://github.com/discourse/discourse-calendar/assets/368961/67a356cd-f80f-4d89-a4e8-2ba5c87280c5">